routes/originGitHub.js

```diff
@@ -62,14 +62,14 @@ router.get(
       const answer = await github.search.repos({
         q: `${repo || ''}+user:${login}+in:name+fork:true`,
         sort: 'stars',
-        per_page: 100
+        per_page: 1000
```

Comment:
Does it actually work? I thought 100 was the maximum?

Reply:
You are right. I've checked the doc and the maximum amount allowed is 100.